damnit/cli.py

```diff
@@ -42,6 +42,11 @@ def excepthook(exc_type, value, tb):
     IPython.start_ipython(argv=[], display_banner=False,
                           user_ns=target_frame.f_locals | target_frame.f_globals | {"__tb": lambda: print(tb_msg)})
 
+def proposal_runs(proposal):
+    proposal_name = f"p{int(proposal):06d}"
+    raw_dir = Path(find_proposal(proposal_name)) / "raw"
+    return set(int(p.stem[1:]) for p in raw_dir.glob("*"))
```

Comment:
We might want to change this check in the future if we can process a run from proc or red even once the raw data is gone. But I think the concept of 'missing data for run X' makes sense, so it shouldn't be a problem to change how we check for that later. :+1: